<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>10</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>296</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Colormap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>90</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Band</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>110</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>60</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>gist_earth</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gist_gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gist_heat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gist_ncar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gist_rainbow </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gist_stern</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gist_yarg </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>autumn </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bone </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cool </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>copper </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>flag </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hsv </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>jet</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pink</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>prism</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>spring</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>summer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>winter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>spectral</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>212</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>216</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>253</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>162</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Lat/Lon</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Row/Col</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>297</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
